playground/backend-integration/vite.config.js

```diff
@@ -22,6 +22,14 @@ function BackendIntegrationExample() {
       entrypoints.push(['bar.css', path.resolve(__dirname, './dir/foo.css')])
 
       return {
+        server: {
+          // same port in playground/test-utils.ts
+          port: 5009,
+          strictPort: true,
+          // TODO: should this be automatically inferred from dev server address?
+          // (see "boolean or string" discussions in https://github.com/vitejs/vite/pull/4337)
```

Comment:
```suggestion
```
There's an issue about this here: https://github.com/vitejs/vite/issues/12597
so let's remove this todo comment

Reply:
Ah, I didn't notice that one. Thanks for the link!